examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -24,16 +24,18 @@
 # Load the behavioral data
 # -------------------------
 import numpy as np
```

Comment:
Do we still need numpy in this example?

Reply:
It's removed, and in `plot_haxby_space_net.py`!